packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts

```diff
@@ -97,7 +97,15 @@ function setupTestSuiteMatrix(
         globalThis['newPrismaClient'] = (...args) => {
           const client = new global['loaded']['PrismaClient'](...args)
           clients.push(client)
-          return client
+          return new Proxy(client.$extends({}), {
+            get(target, prop) {
+              if (prop === '$use' || prop === '$on') {
```

Comment:
That feels a little bit hacky to me. If we are going to keep this permanently, would it be better to skip the tests for `use` and `prop` permanently?